### ISSUE TITLE
Add Support for querying/returning structured facts

### ISSUE THREAD

**Issue by exodusftw** [propyless/panopuppet#137]

**Comment by propyless** [comment]
Hey long time there @exodusftw,
I suppose you mean in the query builder and also returning by getting a nice view of hashes and stuff in the csv file?

**Comment by exodusftw** [comment]
@propyless yeah - it see that you've already defined the fact-contents endpoint of the v4 pdb api, but because of the way it splits the field and expects simple fact names in the export to csv fact field, you can't pass the complex structured fact path tree

**Comment by exodusftw** [comment]
and @propyless  - it's been too long 😞

**Comment by propyless** [comment]
Since its difficult to catch you, timezones and all, I know you popped into the IRC channel a month or so ago, but I've got a gitter channel now, offline chat etc so its easy to respond when your not there.

**Comment by exodusftw** [comment]
you should join the Puppet Community Slack also...gitter is great, unless you're behind a big ugly firewall....
